Robust.Client/UserInterface/Controls/LineEdit.cs

```diff
@@ -344,6 +344,15 @@ protected internal override void KeyBindDown(GUIBoundKeyEventArgs args)
                         InsertAtCursor(clipboard.GetText());
                     }
                 }
+                else if (args.Function == EngineKeyFunctions.TextDelete)
+                {
+                    if (_cursorPosition + 1 > _text.Length || !Editable)
```

Comment:
This is probably a bit clearer.

```suggestion
                    if (_cursorPosition >= _text.Length || !Editable)
```

Reply:
yup, you're right. applied